pages/index.tsx

```diff
@@ -67,11 +68,21 @@ const ManageCandyMachines: NextPage = () => {
                         </Button>
                     </div>
                 ) : !accounts?.length ? (
-                    <span className='mt-5'>
+                    <span className='mt-5 d-flex flex-row flex-items-center '>
```

Comment:
It is a good idea to make this change:
```suggestion
                    <div className='mt-5 d-flex flex-column flex-md-row flex-items-start flex-md-items-center '>
                        <div className='mr-2'>You have no candy machine accounts.</div>
                        <Button
                            className='color-bg-transparent text-semibold'
                            variant='invisible'
                            onClick={() => setIsOpen(true)}
                            sx={{ textTransform: 'capitalize', padding: '0' }}
                        >
                            Create one?
                        </Button>
                        {isOpen && (
                            <Popup onClose={() => setIsOpen(false)} title='Create Candy Machine' size='large'>
                                <CreateCandyMachine />
                            </Popup>
                        )}
                    </div>
```